HW04.Rmd

```diff
@@ -1,126 +1,239 @@
-
-```{r month_chart}
-
-```
-
-## Generating a bar chart that identifies the number of gun deaths associated with each type of intent cause of death. The bars should be sorted from highest to lowest values.
-
-```{r intent}
-
-```
-
-## Generating a boxplot visualizing the age of gun death victims, by sex. Print the average age of female gun death victims.
-
-```{r age_&_sex}
-
-```
-
-
-## How many white males with at least a high school education were killed by guns in 2012?
-
-```{r education}
-
-```
-
-
-## Which season of the year has the most gun deaths?
-
-Assume that:
-
-* Winter = January-March
-* Spring = April-June
-* Summer = July-September
-* Fall = October-December
-
-```{r season}
-
-```
-
-
-# These are more open ended questions from here on out, you could look at making multiple summaries or types of plots. 
-
-### Are whites who are killed by guns more likely to die because of suicide or homicide? How does this compare to blacks and hispanics?
-
-```{r race}
-
-```
-
-(hint maybe looking at percentages would be good)
-```{r race_2}
-
-```
-
-
-### Are police-involved gun deaths significantly different from other gun deaths? Assess the relationship between police involvement and age, police involvement and race, and the intersection of all three variables.
-
-```{r police}
-
-```
-
-Think about performing stats such as a t.test to see if differences you are plotting are statistically significant
-
-```{r stats?}
-
-
-```
-
-
-```{r police_2}
-
-```
-
-```{r stats_2?}
-
-
-```
-
-
-Finally, all together now:
-```{r police_3}
-
-```
-
-
-## Session info
-
-```{r}
-# always good to have this for reproducibility purposes
-devtools::session_info()
-```
-
+---
+title: "Exploring Gun Deaths in America"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning=FALSE, message=FALSE)
+```
+
+## Objectives
+* To explore gun death data curated by FiveThirtyEight [original data available here](https://github.com/fivethirtyeight/guns-data) by
+    * Month
+    * Intent
+    * Age and sex
+    * Education
+    * Season
+    * Intent by race
+    * Police-relatedness 
+
+## Read in the data
+
+The gun_deaths.csv is provided in the repository. 
+```{r load-data, message=FALSE, warning=FALSE}
+library(tidyverse)    # load tidyverse packages, including ggplot2
+library(viridis)      # color scheme
+library(scales)       # required for overlaying percentages over bar plots
+
+# set theme
+theme_set(theme_bw())
+
+my_colors <- c("#2a9d8f", "#E76F51", "#E9C46A", "#3a6b7e", "#F4A261")
+
+# read in the data
+gun_deaths <- read.csv('gun_deaths.csv', 
+                       na.strings=c("NotAvailable"), 
+                       colClasses = c("integer", "integer",  
+                                      "factor", "factor",  
```

Comment:
This is smart. Was this your way of converting the factors to integers for certain columns? I see in the guns table, age is a integer already. and in the age/sex section with ggplot, you did not do any modifications to the age column. Nice!